manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -118,6 +118,8 @@ spec:
       metadata:
         name: ptp-operators
         namespace: openshift-ptp
+        labels:
+          openshift.io/cluster-monitoring: "true"
```

Comment:
This shall be added in above Namespace section.

Reply:
Sorry my bad! 

Thanks, done!